src/Modules/CalcPerform.lua

```diff
@@ -2294,17 +2350,41 @@ function calcs.perform(env, avoidCache)
 			end
 		end
 	end
+	local allyCurses = {}
+	local allyPartyCurses = env.build.partyTab.processedInput["Curse"]
+	if allyPartyCurses["Curse"] then
+		allyCurses.limit = allyPartyCurses.limit
+	else	
+		allyPartyCurses = { Curse = {} }
+	end
+	for curseName, curse in pairs(allyPartyCurses["Curse"]) do
+		local newCurse = {
+			name = curseName,
+			priority = 0,
+			modList = new("ModList")
+		}
+		local mult = curse.effectMult / 100
+		if curse.isMark then
+			newCurse.isMark = true
+		else
+			mult = mult * enemyDB:More(nil, "CurseEffectOnSelf")
+		end
+		newCurse.modList:ScaleAddList(curse.modList, mult)
+		t_insert(allyCurses, newCurse)
+	end
```

Comment:
Can we find a way to pull this and the other curse logic out into a shared function?  If boss curse effect ever gets more calculations I'm afraid we'd miss it either here or in the other curse effect calc section.